src/EnergyPlus/DXCoils.cc

```diff
@@ -3972,6 +4172,7 @@ namespace DXCoils {
 						CurveVal = CurveValue( DXCoil( DXCoilNum ).MSCCapFFlow( I ), 1.0 );
 						if ( CurveVal > 1.10 || CurveVal < 0.90 ) {
 							ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + DXCoil( DXCoilNum ).Name + "\", curve values" );
+							ShowContinueError( cAlphaFields( 14 + ( I - 1 ) * 6 ) + " = " + Alphas( 14 + ( I - 1 ) * 6 ) );
```

Comment:
Field name is redundant here and in the next line.  This line is nice, but then I'd remove the field name in the next line.

There's so many of these, all using about six lines each.  Seems ripe for a separate function.
```
CheckCuveAtRatedConditions( RoutineName + CurrentModuleObject, DXCoil( DXCoilNum ).Name, DXCoil( DXCoilNum ).MSCCapFFFlow, cAlphaFields( . . . ), X1, X2-optional, X3-optional );
```